scipy/io/tests/test_mmio.py

```diff
@@ -5,8 +5,8 @@
 import os
 import shutil
 from numpy import array,transpose
-from numpy.testing import TestCase, run_module_suite, assert_array_almost_equal, \
-            assert_equal, rand
+from numpy.testing import TestCase, run_module_suite, assert_equal,\
```

Comment:
If you can be bothered, we usually prefer parentheses to using the backslash for line continuation.